apps/web/pages/api/book/request-reschedule.ts

```diff
@@ -209,49 +220,53 @@ const handler = async (
         rescheduleLink: builder.rescheduleLink,
       });
 
-      const evt: CalendarEvent = {
-        title: bookingToReschedule?.title,
-        type: event && event.title ? event.title : bookingToReschedule.title,
-        description: bookingToReschedule?.description || "",
-        customInputs: isPrismaObjOrUndefined(bookingToReschedule.customInputs),
-        startTime: bookingToReschedule?.startTime ? dayjs(bookingToReschedule.startTime).format() : "",
-        endTime: bookingToReschedule?.endTime ? dayjs(bookingToReschedule.endTime).format() : "",
-        organizer: userOwnerAsPeopleType,
-        attendees: usersToPeopleType(
-          // username field doesn't exists on attendee but could be in the future
-          bookingToReschedule.attendees as unknown as PersonAttendeeCommonFields[],
-          tAttendees
-        ),
-        uid: bookingToReschedule?.uid,
-        location: bookingToReschedule?.location,
-        destinationCalendar:
-          bookingToReschedule?.destinationCalendar || bookingToReschedule?.destinationCalendar,
-        cancellationReason: `Please reschedule. ${cancellationReason}`, // TODO::Add i18-next for this
-      };
+      if (bookingToReschedule.userId) {
```

Comment:
nit: This check could also be moved very early in the code and we can return early